### DISCUSSION TITLE
请教一下大佬如何修改或删除ytdlp这个音源

### DISCUSSION THREAD

**Discussion by DDHZJ** [1715173329/UnblockNeteaseMusic#1332] [Q&A] [answered]
播放音乐弹出这个提示 You must install "yt-dlp" before using the "ytdlp" source.
小白找了很多方法（删除ytdlp这个音乐或者改其他音源啥的），但是都失败了
![QQ截图20231225200523](https://github.com/UnblockNeteaseMusic/server/assets/61381136/5d2ad375-6173-4de5-80b3-2cfd65f1fad8)

**Comment by DDHZJ** [comment]
摸索到这一步了😂VIP的歌放不出来咋办
![QQ截图20231225213209](https://github.com/UnblockNeteaseMusic/server/assets/61381136/b21561a7-2598-4bf4-a732-72012704d549)

**Reply by pan93412** [reply, marked as answer]
YouTube 要掛全局代理或參考 https://github.com/UnblockNeteaseMusic/server/issues/1355
